flow360/component/flow360_params/flow360_params.py

```diff
@@ -119,6 +119,152 @@ class MeshBoundary(Flow360BaseModel):
+    )
+
+
+class TurbulentLengthScale(Flow360BaseModel):
+    """
+    turbulentLengthScale : non-dimensional [`L_gridUnit`]
+        The turbulent length scale is an estimation of the size of the eddies that are modeled/not resolved.
+        Applicable only when using SST model. This is related to the turbulent kinetic energy and turbulent
+        specific dissipation rate as: `L_T = sqrt(k)/(pow(beta_0^*, 0.25)*w)` where `L_T` is turbulent length scale,
+        `k` is turbulent kinetic energy, `beta_0^*` is 0.09 and `w` is turbulent specific dissipation rate.
+        Applicable only when using SST model.
+    """
+
+    turbulent_length_scale: Optional[PositiveFloat] = pd.Field(alias="turbulentLengthScale")
+
+
+class ModifiedTurbulentViscosityRatio(Flow360BaseModel):
+    """
+    modifiedTurbulentViscosityRatio : non-dimensional [`-`]
+        The ratio of modified turbulent eddy viscosity (SA) over the freestream viscosity.
+        Applicable only when using SA model.
+    """
+
+    modified_turbulent_viscosity_ratio: Optional[PositiveFloat] = pd.Field(
+        alias="modifiedTurbulentViscosityRatio"
+    )
+
+
+class ModifiedTurbulentViscosity(Flow360BaseModel):
+    """
+    modifiedTurbulentViscosity : non-dimensional [`C_inf*L_gridUnit`]
+        The modified turbulent eddy viscosity (SA). Applicable only when using SA model.
+    """
+
+    modified_turbulent_viscosity: Optional[PositiveFloat] = pd.Field(
+        alias="modifiedTurbulentViscosity"
+    )
+
+
+# pylint: disable=missing-class-docstring
+class SpecificDissipationRateAndTurbulentKineticEnergy(
+    SpecificDissipationRate, TurbulentKineticEnergy
+):
+    pass
+
+
+# pylint: disable=missing-class-docstring
+class TurbulentViscosityRatioAndTurbulentKineticEnergy(
+    TurbulentViscosityRatio, TurbulentKineticEnergy
+):
+    pass
+
+
+# pylint: disable=missing-class-docstring
+class TurbulentLengthScaleAndTurbulentKineticEnergy(TurbulentLengthScale, TurbulentKineticEnergy):
+    pass
+
+
+# pylint: disable=missing-class-docstring
+class TurbulentIntensityAndSpecificDissipationRate(TurbulentIntensity, SpecificDissipationRate):
+    pass
+
+
+# pylint: disable=missing-class-docstring
+class TurbulentIntensityAndTurbulentViscosityRatio(TurbulentIntensity, TurbulentViscosityRatio):
+    pass
+
+
+# pylint: disable=missing-class-docstring
+class TurbulentIntensityAndTurbulentLengthScale(TurbulentIntensity, TurbulentLengthScale):
+    pass
+
+
+# pylint: disable=missing-class-docstring
+class SpecificDissipationRateAndTurbulentViscosityRatio(
+    SpecificDissipationRate, TurbulentViscosityRatio
+):
+    pass
+
+
+# pylint: disable=missing-class-docstring
+class SpecificDissipationRateAndTurbulentLengthScale(SpecificDissipationRate, TurbulentLengthScale):
+    pass
+
+
+# pylint: disable=missing-class-docstring
+class TurbulentViscosityRatioAndTurbulentLengthScale(TurbulentViscosityRatio, TurbulentLengthScale):
+    pass
+
+
+AllowedTurbulenceSpecifications = Union[
```

Comment:
Lets call this type the same as the parameter: `TurbulenceQuantitiesType`

Reply:
Done